cfe_internal/enterprise/federation/federation.cf

```diff
@@ -88,6 +102,12 @@ bundle agent transport_user
       depth_search => default:recurse_with_base("inf"),
       file_select => default:all,
       perms => default:mog( "600", $(user), "root" );
+    "$(home)/.ssh/authorized_keys"
+      create => "true",
+      edit_template_string => "{{#-top-}}{{{.}}}$(const.n){{/-top-}}",
+      template_data => @(pubkeys),
```

Comment:
`pubkeys` is defined in `bundle agent config`, so I believe it needs to either be passed to this `bundle agent transport_user` as a parameter, or it needs to be referenced using it's fully qualified name `config.pubkeys` or `cfengine_enterprise_federation:config.pubkeys`.